### ISSUE TITLE
[STUDIO][1.0.0-beta.3] Acteur liste de bouton couleur 

### ISSUE THREAD

**Issue by WITNicolas** [witsa/synapps#368]
La couleur du texte (non sélectionné) n'est a priori pas prise en compte. Le fond fonctionne très bien par contre.
![image](https://user-images.githubusercontent.com/33632664/110206499-53181b80-7e7e-11eb-83d0-5a076d0f3f09.png)

**Comment by Raphwit** [comment]
Effectivement, le "mode" prend le dessus sur la couleur du texte, c'est à corriger si le comportement n'est pas celui attendu !
